lib/assets/javascripts/turbograft/remote.coffee

```diff
@@ -51,10 +51,13 @@ class TurboGraft.Remote
 
   createPayload: (form) ->
     if form
-      if form.querySelectorAll("[type='file']").length > 0
-        formData = new FormData(form)
+      cleanForm = form.cloneNode(true)
+      for node in cleanForm.querySelectorAll('input:not([name])')
```

Comment:
^ Red herring, but still a change that should probably happen

> cleanForm.querySelector("#order-note").value
> => "test"

That's the original value from page init.  Perhaps something is just messed up with cloneNode and textareas?